pkgs/tools/networking/openconnect/default.nix

```diff
@@ -1,12 +1,39 @@
-{ lib, stdenv, fetchurl, pkg-config, openssl ? null, gnutls ? null, gmp, libxml2, stoken, zlib, fetchgit, darwin } :
+{ lib, stdenv, fetchurl, fetchgit
+, pkg-config, makeWrapper,
```

Comment:
```suggestion
, pkg-config, makeWrapper,
```

Reply:
GitHub says "This suggestion is invalid because no changes were made to the code."